python/ibmos2spark/osconfig.py

```diff
@@ -170,3 +170,48 @@ def __init__(self, sparkcontext, credentials, name=None, public=False, swift2d_d
 
   def url(self, container_name, object_name):
     return swifturl2d(self.name, container_name, object_name)
+
+
+class CloudObjectStorage(object):
+
+    def __init__(self, sparkcontext, bucket_name, credentials, driver='com.ibm.stocator.fs.ObjectStoreFileSystem'):
+
+        '''
+        sparkcontext:  a SparkContext object.
+
+        bucket_name (projectId in DSX):  string that identifies the bucket you want
+            to access files from in the COS service instance.
+            In DSX, bucket_name is the same as projectId. One bucket is
+            associated with one project.
+
+        credentials:  a dictionary with the following required keys:
+          * endpoint
+          * access_key
+          * secret_key
+
+        When using this on DSX credentials and bucket_name can be obtained
+        in DSX - Notebooks by clicking on the datasources palette then
+        choose the datasource you want to access then hit insert credentials.
+
+        '''
+
+        self.bucket_name = bucket_name
+
+        # check if all required values are availble
+        credential_key_list = ["endpoint", "access_key", "secret_key"]
+
+        for i in range(len(credential_key_list)):
+            key = credential_key_list[i]
+            if (not key in credentials):
+                raise ValueError("Invalid input: credentials.{} is required!".format(key))
+
+        # setup config
+        prefix = "fs.s3d.service"
+        hconf = sparkcontext._jsc.hadoopConfiguration()
+        hconf.set(prefix + ".impl", driver)
```

Comment:
@bassel-zeidan This key suppose to be already setup in core-sites.xml. There is no need to setup it again.